R/bayes_analysis.R

```diff
@@ -230,226 +232,260 @@ relative_rank_text <- function(results) {
-                            '</br> Deviance from UME model:',round(X5.y, digits=2)))}
-  if (ncol(c)>6) { 
-    p=p %>% 
-      add_trace(data=all,
-                x=~X6.x, y=~X6.y, type='scatter', mode='markers', 
-                marker=list(size=4, color = '#CAEFD1',
-                            line = list(color = 'rgb(0,128,0)',
-                                        width = 2)),
-                hoverinfo='text',
-                text=~paste('</br>', all$names,
-                            '</br> Arm 6',
-                            '</br> Deviance from NMA model:',round(X6.x, digits=2),
-                            '</br> Deviance from UME model:',round(X6.y, digits=2)))}
-  progress$inc(0.2, detail="Exporting results")
-  list(p=p,y=y)
+                x = ~X4.x, y = ~X4.y, type = 'scatter', mode = 'markers', 
+                marker = list(size = 4,
+                              color = '#CAEFD1',
+                              line = list(color = 'rgb(0,128,0)',
+                                        width = 2)
+                              ),
+                hoverinfo = 'text',
+                text = ~paste('</br>', all$names,
+                              '</br> Arm 4',
+                              '</br> Deviance from NMA model:', round(X4.x, digits = 2),
+                              '</br> Deviance from UME model:', round(X4.y, digits = 2)))}
+  if (ncol(c) > 5) { 
+    p = p %>% 
+      add_trace(data = all,
+                x = ~X5.x, y = ~X5.y, type = 'scatter', mode = 'markers',
+                marker = list(size = 4,
+                              color = '#CAEFD1',
+                              line = list(color = 'rgb(0,128,0)',
+                                        width = 2)
+                              ),
+                hoverinfo = 'text',
+                text = ~paste('</br>', all$names,
+                              '</br> Arm 5',
+                              '</br> Deviance from NMA model:', round(X5.x, digits = 2),
+                              '</br> Deviance from UME model:', round(X5.y, digits = 2)))}
+  if (ncol(c) > 6) { 
+    p = p %>% 
+      add_trace(data = all,
+                x = ~X6.x, y = ~X6.y, type = 'scatter', mode = 'markers', 
+                marker = list(size = 4,
+                              color = '#CAEFD1',
+                              line = list(color = 'rgb(0,128,0)',
+                                          width = 2)
+                              ),
+                hoverinfo = 'text',
+                text = ~paste('</br>', all$names,
+                              '</br> Arm 6',
+                              '</br> Deviance from NMA model:', round(X6.x, digits = 2),
+                              '</br> Deviance from UME model:', round(X6.y, digits = 2)))}
+  progress$inc(0.2, detail = "Exporting results")
+  list(p = p, y = y)
 }
 
 
 
-### Per-arm residual deviance 
-
-stemplot.df <- function(c,x) {
+#' Per-arm residual deviance
```

Comment:
This one doesn't cover what the function does by my understanding. The comment doesn't inform that the function creates a plot, rather than a calculation.